dev/scripts/exglobal_enkf_earc_vrfy.py

```diff
@@ -14,37 +15,20 @@ def main():
 
     config = cast_strdict_as_dtypedict(os.environ)
 
-    # Instantiate the Archive object
+    # Instantiate the Archive task object
     archive = Archive(config)
 
-    # Pull out all the configuration keys needed to run the rest of archive steps
-    keys = ['current_cycle', 'RUN', 'PSLOT', 'ROTDIR', 'PARMgfs',
-            'ARCDIR', 'MODE', 'DO_JEDIATMENS', 'DO_FIT2OBS', 'DO_JEDIATMVAR',
-            'DO_JEDISNOWDA', 'DO_AERO_ANL', 'DO_PREP_OBS_AERO', 'NET', 'MODE', 'FHOUT_GFS',
-            'FHMAX_HF_GFS', 'FHOUT_GFS', 'FHMAX_FITS', 'FHMAX', 'FHOUT', 'FHMAX_GFS', 'DO_GSISOILDA', 'DO_LAND_IAU']
+    # Collect all archive variables in complete arch_dict for YAML templates
+    # Use static utility methods from ArchiveVrfyVars
+    arch_dict = ArchiveVrfyVars.get_all_yaml_vars(archive.task_config)
 
-    archive_dict = AttrDict()
-    for key in keys:
-        archive_dict[key] = archive.task_config.get(key)
-        if archive_dict[key] is None:
-            print(f"Warning: key ({key}) not found in task_config!")
+    # Pass arch_dict to configure_vrfy which will render the Jinja2 YAML
+    arcdir_set = archive.configure_vrfy(AttrDict(arch_dict))
```

Comment:
```suggestion
    arcdir_set = archive.configure_vrfy(arch_dict)
```
Same comment as above.